subsys/bluetooth/host/iso.c

```diff
@@ -956,21 +902,25 @@ static struct net_buf *hci_le_set_cig_params(struct bt_iso_create_param *param)
 		}
 
 		if (!qos->tx) {
-			/* Use RX PHY if TX is not set (disabled) */
-			cis->m_phy = qos->rx->phy;
+			/* Use RX PHY if TX is not set (disabled)
+			 * to avoid setting invalid values
+			 */
+			cis_param->m_phy = qos->rx->phy;
```

Comment:
qos->rx may be be NULL (see above), but it is still being dereferenced.

Reply:
We have a check 
```
		if (!qos->tx && !qos->rx) {
```

This line is only executed if `if (!qos->tx) {`, so `qos->rx` will always be non-NULL at this point.